config/locales/mailers/en.yml

```diff
@@ -74,6 +74,9 @@ en:
           addressed: Hi, %{name}!
           unaddressed: Hi!
         introductory: Hello from the Archive of Our Own!
+        tag_wrangling_supervisor:
+          html: Hello %{tag_wrangler_supervisors},
```

Comment:
Would it be okay to use the standard informal greeting instead? That would be `Hi, Tag Wrangler Supervisors!`. I didn't want to make the email different from the Jira issue, but it seems a bit awkward to have two so similar greetings.

Reply:
I checked with TW and the standard one is indeed fine! They also send their thanks for working on this; they've very excited.